string-manipulation-1/src/capitalize.js

```diff
@@ -0,0 +1,35 @@
+/* exported capitalize */
+
+function capitalize(word) {
+  var wordResult = '';
+  for (var i = 0; i < word.length; i++) {
+    if (i === 0) {
+      if (word[i].charCodeAt(0) >= 97) {
+        var capAsci = word[i].charCodeAt(0) - 32;
+        wordResult += String.fromCharCode(capAsci);
+      } else {
+        wordResult += word[i];
+      }
+    } else {
+      if (word[i].charCodeAt(0) <= 90) {
+        var smallAsci = word[i].charCodeAt(0) + 32;
+        wordResult += String.fromCharCode(smallAsci);
+      } else {
+        wordResult += word[i];
+      }
```

Comment:
You can simplify this by quite a bit with the `toUpperCase` and `toLowerCase` methods. It's important to know all the details of what's occurring in your code, but for something as nitty-gritty as converting characters, we can avoid doing character code calculations entirely by making use of those methods. Godo that you know how to do this, though!